src/pages/tutorials/working-with-feature-layers.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Working with feature layers
+description: Learn how to [the content] in Leaflet.
+layout: tutorials.hbs
+---
+
+# {{ page.data.title }}
+
+{{ page.data.description }}
+
+This tutorial assumes you know how to create a basic Leaflet map.  To learn how to do this, please visit the [Create your first app](http://esri.github.io/esri-leaflet/tutorials/create-your-first-app.html "Create your first app") tutorial.
+
+##### Copy Starter HTML Page
+
+Start by copying this skeleton code which outlines the structure of a single page website and save it in your own new `.html` file.  The code includes a `<script>` that constructs the Leaflet map object and adds a basemap from ArcGIS Online.
+
+```xml
+<!doctype html>
+<html lang="en">
+<head>  
+  <meta charset="utf-8">
+  <title>Leaflet Map with a Feature Layer</title>  
+  <meta name='viewport' content='initial-scale=1,maximum-scale=1,user-scalable=no' />
+
+  <!-- Load Leaflet from CDN-->
+  <link rel="stylesheet" href="https://npmcdn.com/leaflet@0.7.7/dist/leaflet.css" />
+  <script src="https://npmcdn.com/leaflet@0.7.7/dist/leaflet.js"></script>
+
+  <!-- Load Esri Leaflet from CDN -->
+  <script src="https://cdn.jsdelivr.net/leaflet.esri/1.0.4/esri-leaflet.js"></script>
+
+  <style>
+    html,
+    body,
+    #map {
+      height: 100%;
+      width: 100%;
+      margin: 0;
+      padding: 0;
+    }
+  </style>
+</head>
+<body>    
+    <div id="map"></div>
+    
+    <script>
+        var map = L.map('map', {
+          center: [45.526, -122.667],
+          zoom: 13
+        });
+        
+        var esriStreets = L.esri.basemapLayer('Streets').addTo(map);    
+    </script>    
+</body>
+</html>
+```
+
+##### Add a Feature Layer to the Map
+
+We will add a REST service using the `L.esri.featureLayer` class from the Esri Leaflet plugin.  This can be either a **Map Service** or **Feature Service**.  You must provide the url for the service.  Below is an example for creating a Feature Layer using both types of services:
```

Comment:
@rowanwins is working on another topic to differentiate between different layer types in #73.  linking to that from here would be perfect.